### PR TITLE
[IA-4586] New cached images with notebook 6.5.4

### PR DESCRIPTION
**Pull request by LizBaldo**
__Jira ticket__: https://broadworkbench.atlassian.net/browse/IA-4586

Corresponding terra-docker PR: https://github.com/DataBiosphere/terra-docker/pull/461

### What

-

### Why

-

### What to test

Tested on my BEE that all terra environments are coming online, and that the TNU package is on version 0.13.0

### Who tested and where

- [ ] This change is covered by automated tests 
  - _NB: Rerun automation tests on this PR by commenting `jenkins retest` or `jenkins multi-test`._
- [ ] I validated this change 
- [ ] Primary reviewer validated this change 
- [ ] I validated this change in the dev environment

### CODE REVIEW

**Comment by LizBaldo** [comment]
jenkins retest

File: jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh [hunk 1]
```diff
@@ -16,12 +16,12 @@ set -e -x
...
-terra_jupyter_bioconductor="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-bioconductor:2.2.1"
-terra_jupyter_hail="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-hail:1.1.1"
-terra_jupyter_gatk="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.3.1"
-terra_jupyter_aou="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.2.1"
+terra_jupyter_python="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:1.1.3"
+terra_jupyter_r="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:2.2.3"
+terra_jupyter_bioconductor="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-bioconductor:2.2.3"
+terra_jupyter_hail="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-hail:1.1.3"
+terra_jupyter_gatk="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.3.3"
+terra_jupyter_aou="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.2.3"
```
Comment (yonghaoy) [inline jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh]: can you leave `leonardo.aouImageUrl` to `2.2.1` for now?

File: jenkins/gce-custom-images/prepare_gce_image.sh [hunk 2]
```diff
@@ -16,11 +16,11 @@ set -e -x
...
-terra_jupyter_python="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:1.1.1"
-terra_jupyter_r="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:2.2.1"
-terra_jupyter_bioconductor="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-bioconductor:2.2.1"
-terra_jupyter_gatk="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.3.1"
-terra_jupyter_aou="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.2.1"
+terra_jupyter_python="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:1.1.3"
+terra_jupyter_r="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:2.2.3"
+terra_jupyter_bioconductor="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-bioconductor:2.2.3"
+terra_jupyter_gatk="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.3.3"
+terra_jupyter_aou="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.2.3"
```
Comment (yonghaoy) [inline jenkins/gce-custom-images/prepare_gce_image.sh]: ditto

**Review by yonghaoy — APPROVED** [review]

**Comment by LizBaldo** [comment]
jenkins retest

**Review by tlangs — APPROVED** [review]

**Comment by LizBaldo** [comment]
jenkins retest

**Pull request merged**